lib/generators/manageiq/plugin/plugin_generator.rb

```diff
@@ -105,6 +126,26 @@ def plugin_description
       @plugin_description ||= "#{plugin_short_name.titleize} plugin for #{Vmdb::Appliance.PRODUCT_NAME}."
     end
 
+    def node_version
+      @node_version ||= package_json.dig("engines", "node")
+    end
+
+    def npm_version
+      @npm_version ||= package_json.dig("engines", "npm")
+    end
+
+    def yarn_version
+      @yarn_version ||= package_json.dig("engines", "yarn") || ">= 0.20.1"
+    end
+
+    def package_managaer
+      @package_managaer ||= package_json["packageManager"]
```

Comment:
This should really come from the committed yarn/releases/yarn-*.cjs file, because they have to match.  Perhaps this can be something roughly like:

```ruby
File.basename(Dir.glob(".yarn/releases/yarn-*.cjs").first).split(/-./).first(2).join("@")
```